proposals/003-perf-book.md

```diff
@@ -0,0 +1,69 @@
+# Haskell Performance Tuning Book
+
+##  Introduction
+
+Performance matters, and Haskell’s runtime and memory models are sufficiently different from traditional programming languages to require additional knowledge, which isn’t commonly covered by most learning resources, in order to effectively debug and tune the performance of Haskell software.
+
+This proposal aims to bridge this gap by creating a learning resource in the form of a book describing how to write fast idiomatic Haskell, and how to debug performance issues and tune Haskell programs.
+
+## Goals
+
+We would like to create a **freely available online book** which:
+
+- Is relatively **short**
+- Can be **maintained over time**, to make sure this knowledge remains accurate and up-to-date
+- Targets **hobbyists and professional** Haskell developers
+- Focuses on **real world** usage and techniques
+
+These goals lead to the following suggestions:
+
+- Writing this book should be a **community-driven** effort
+- This book should focus on writing **idiomatic Haskell** code that is fast rather than advanced state-of-the-art techniques
+- The book should cover the **principles and mindset** of performant Haskell code as well as provide concrete examples of debugging and tuning the performance of a Haskell project using **case studies**
+
+## Motivation
+
+One motivation for this book is that it can help **drive the adoption** of Haskell in professional settings by providing knowledge regarding what to do when debugging and tuning performance is necessary, and help **improve the confidence** of practitioners and would-be practitioners.
+
+## Prior Art (if applicable)
+
+[Haskell High Performance Programming / Samuli Thomasson (2016, Packt)](https://www.packtpub.com/product/haskell-high-performance-programming/9781786464217)
+
+## People
+
+The following roles are necessary to make this project successful:
+
+- **Subject matter expert**(s): directing the “spirit of the book” and/or sharing knowledge with writers
+- **Writer**(s): writing the content of the book
+- **Reviewer**(s): providing feedback and corrections
+- **Editor**(s): quality assurance for the book
+
+## Timeline
+
+## Outcomes
```

Comment:
Outcomes should include 

1. A shiny new book, available in <formats> and accessible via <people>, maintained by <maintainers>
2. All of the accoutrements of building the book itself (a repo + instructions is a deliverable!)
3. Content that handles <subjects>
4. Resulting in a <citable resource that makes our brains bigger>

Reply:
I would love this section to include a description of how the community would access the book. I presume it would just be hosted on a website somewhere. But it would be nice to say.

Reply:
I've added a few points with your feedback.